src/Jackett.Common/Jackett.Common.csproj

```diff
@@ -1,26 +1,62 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>net8.0;netstandard2.0</TargetFrameworks>
     <Version>0.0.0</Version>
     <LangVersion>9</LangVersion>
     <NoWarn />
     <WarningsAsErrors />
     <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Release|net8.0|AnyCPU'">
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
+    <WarningsAsErrors />
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|net8.0|AnyCPU'">
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
+    <WarningsAsErrors />
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
+    <SQLitePCLRawLibESQLite3x64Path>$(NuGetPackageRoot)sqlitepclraw.lib.e_sqlite3\2.1.9\runtimes\linux-x64\native\libe_sqlite3.so</SQLitePCLRawLibESQLite3x64Path>
+    <SQLitePCLRawLibESQLite3x86Path>$(NuGetPackageRoot)sqlitepclraw.lib.e_sqlite3\2.1.9\runtimes\linux-x86\native\libe_sqlite3.so</SQLitePCLRawLibESQLite3x86Path>
+  </PropertyGroup>
+
+  <Target Name="CopySQLiteLib" AfterTargets="Build;Test" Condition="'$(TargetFramework)' == 'netstandard2.0'">
+    <Copy SourceFiles="$(SQLitePCLRawLibESQLite3x64Path)" DestinationFolder="$(OutDir)runtimes/linux-x64/native/" />
+    <Copy SourceFiles="$(SQLitePCLRawLibESQLite3x86Path)" DestinationFolder="$(OutDir)runtimes/linux-x86/native/" />
+  </Target>
+
+  <!-- Conditionally obtain references for the .NET 8.0 target -->
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net8.0' ">
+    <PackageReference Include="SQLitePCLRaw.bundle_e_sqlite3" Version="2.1.9" />
+  </ItemGroup>
+
+  <!-- Conditionally obtain references for the .NET462 target -->
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
+    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.3" />
+    <PackageReference Include="SQLitePCLRaw.provider.e_sqlite3" Version="2.1.9" />
+    <PackageReference Include="SQLitePCLRaw.lib.e_sqlite3" Version="2.1.9" />
+  </ItemGroup>
+  
```

Comment:
The current "Master" branch of Jackett on .NET Framework 4.6.2 has always run in 32-bit mode (x86). Accordingly, when it was run on Linux x64 using Mono, it expected the x64 `libe_sqlite3.so` library. A .NET Framework 4.6.2 application doesn't know how to look for libraries in the runtimes folder, only in the application's root folder. I fixed the issue with JacketConsole so that it can now run in both x86 and x64 modes, depending on the OS. Additionally, I opened an issue (https://github.com/ericsink/SQLitePCL.raw/issues/604) because this library does not work correctly on Mono and cannot find the `libe_sqlite3.so` library (If this worked, there would be no need for this copying). Therefore, depending on whether it's x86 or x64, the appropriate library has to be copied into the runtimes folder, and then during execution, it must be copied to the root folder. Oh, and I forgot to mention that when using Dapper, there were errors with the old version of `Microsoft.Data.Sqlite.Core=2.2.0`

Reply:
If I remember correctly, the issue wasn't with Dapper, but rather that the `Microsoft.Data.Sqlite=2.2.0` version was throwing the error `'SQLite Error 1: "near 'ON': syntax error"'` in the SQLite query on the line '`ON CONFLICT(TrackerId) DO UPDATE SET`' — it didn't recognize this syntax.

In .NET Framework 4.6.2 on Linux with Mono, specifying `Microsoft.Data.Sqlite=2.2.0` will require us to manually download and place the `libe_sqlite3.so` file in the root directory, depending on our OS (`linux-x86` or `linux-x64`). So why should we use the old version when we are using the new one on the .NET 8 platform? Or did I misunderstand the Microsoft.Data.Sqlite=2.2.0?